cordapp/src/main/kotlin/net/corda/plugins/CordappPlugin.kt

```diff
@@ -79,6 +99,26 @@ class CordappPlugin : Plugin<Project> {
         jarTask.dependsOn(task)
     }
 
+    private fun addContractDependenciesListToJar(project: Project, directNonCordaDependencies: MutableSet<File>) {
+        val hashes = calculateContractDependencyHashes(project)
+        val cordInfFolder = File(project.buildDir, "CORDA-DEPS")
+        cordInfFolder.mkdir()
+        val backingFile = File(cordInfFolder, "deps.zip")
```

Comment:
Doesn't Gradle contain functionality to do this kind of thing already? _It makes JARs..._